docs/docusaurus.config.js

```diff
@@ -119,6 +119,63 @@ const config = {
                             },
                         ],
                     },
+                    {
+                        title: 'How-to guides',
+                        items: [
+                            {
+                                label: 'Angular Feature Flags',
+                                href: 'https://www.flagsmith.com/blog/angular-feature-flag',
+                            },
+                            {
+                                label: 'Python Feature Flags',
+                                href: 'https://www.flagsmith.com/blog/python-feature-flag',
+                            },
+                            {
+                                label: 'Java Feature Flags',
+                                href: 'https://www.flagsmith.com/blog/java-feature-toggle',
+                            },
+                            {
+                                label: 'Ruby Feature Flags',
+                                href: 'https://www.flagsmith.com/blog/ruby-feature-flags',
+                            },
+                            {
+                                label: 'Javascript Feature Flags',
```

Comment:
```suggestion
                                label: 'JavaScript Feature Flags',
```

JavaScript is the correct capitalisation :)